File: Architecture.Core/Entities/ProfEducationInfo.cs
using Architecture.Core.Entities.Core;
using Architecture.Core.Entities.LU;
using System;

namespace Architecture.Core.Entities
{
    public class ProfEducationInfo : Auditable
    {
        public int EducationInfoId { get; set; }
        public int ProfileId { get; set; }
        public virtual ProfBasicInfo ProfBasicInfo { get; set; }
        public int? DegreeTypeId { get; set; }
        public virtual DegreeType DegreeType { get; set; }
        public string InstitutionName { get; set; }     
        public DateTime StartYear { get; set; }
        public DateTime EndYear { get; set; }
        public string UniversityAddress { get; set; }
        public string ActivitiesAndSocieties { get; set; }
        public string Result { get; set; }
    }
}

File: Architecture.Web/ClientApp/src/app/Shared/Entity/ClientProfile/profEducationInfo.ts
import { Data } from '@angular/router';
import { Auditable } from '../Common/auditable';

export class profEducationInfo extends Auditable {
    public educationInfoId: number;
    public profileId: number;
    //public virtual ProfBasicInfo ProfBasicInfo { get; set; }

    public degreeTypeId: number;
    //    public virtual degreeType DegreeType { get; set; }

    public degreeType: string;

    public institutionName: string;

    public universityAddress: string

    public activitiesAndSocieties: string

    public result: string

    public startYear: Date;

    public endYear: Date;

}


